### ISSUE TITLE
TypeError: 'NoneType' object is not subscriptable

### ISSUE THREAD

**Issue by mikix** [Machine-Learning-for-Medical-Language/cnlp_transformers#106]
When running some real hospital notes against the cnlpt negation endpoint, I occasionally get a 500 error. This is the server log from one of those calls.

I don't have the text right now, I can work on getting that, if I can clean it safely.

```
WARNING:Negation_REST_Processor:Received document of len 781 to process with 0 entities
The following columns in the test set don't have a corresponding argument in `CnlpModelForClassification.forward` and have been ignored: text. If text are not expected by `CnlpModelForClassification.forward`,  you can safely ignore this message.
***** Running Prediction *****
  Num examples = 0
  Batch size = 8
INFO:     172.17.0.3:56486 - "POST /negation/process HTTP/1.1" 500 Internal Server Error
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/uvicorn/protocols/http/httptools_impl.py", line 376, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/usr/local/lib/python3.10/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/usr/local/lib/python3.10/site-packages/fastapi/applications.py", line 208, in __call__
    await super().__call__(scope, receive, send)
  File "/usr/local/lib/python3.10/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/usr/local/lib/python3.10/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/usr/local/lib/python3.10/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/usr/local/lib/python3.10/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc
  File "/usr/local/lib/python3.10/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.10/site-packages/starlette/routing.py", line 656, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.10/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.10/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/usr/local/lib/python3.10/site-packages/fastapi/routing.py", line 226, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.10/site-packages/fastapi/routing.py", line 159, in run_endpoint_function
    return await dependant.call(**values)
  File "/usr/local/lib/python3.10/site-packages/cnlpt/api/negation_rest.py", line 65, in process
    predictions = output.predictions[0]
TypeError: 'NoneType' object is not subscriptable
```

**Comment by angus-lherrou** [comment]
Hi Michael, what version of cnlp-transformers do you have installed?

**Comment by mikix** [comment]
From pip the other day: 0.4.0

**Comment by mikix** [comment]
This looks like it is entirely due to passing in 0 entities. I can work around this by just not calling cnlpt in that edge case. But maybe it's also nicer if cnlpt did an early exit as well.

**Comment by angus-lherrou** [comment]
Thanks for the feedback! We will add a catch for that and emit a better error.

**Comment by angus-lherrou** [comment]
The fix for this is in #112 and will be released in v0.5.0. Instead of emitting an error we just return an empty list if zero entities are passed in.
